community/cypher/cypher/src/test/scala/org/neo4j/cypher/CartesianProductNotificationAcceptanceTest.scala

```diff
@@ -99,9 +103,22 @@ class CartesianProductNotificationAcceptanceTest extends CypherFunSuite with Gra
     verify(logger, never) += any()
   }
 
-  private def createCompiler() =
+  private def createCompiler() = {
+
+    val entityAccessor = new EntityAccessor {
```

Comment:
This class is copied a bunch of times into different test files. Why not create a single one and share it between the tests?